### ISSUE TITLE
No comment syntax in mld files

### ISSUE THREAD

**Issue by lpw25** [ocaml/odoc#998]
I think there is currently no syntax for comments in odoc's syntax. This is fine within documentation comments, but less fine in `.mld` files.

**Comment by sanette** [comment]
Indeed, I have auto-generated mld files, so I would like to start these files with a line
```# auto-generated file```
but I don't know how to do this

**Comment by panglesd** [comment]
As a work-around, you can use comments in html-specific backend:
```
{%html: <!-- auto-generated file --> %}
```
or as @lpw25 suggested in another channel, you can abuse the backend specific syntax, which allows for non-existing backends:
```
{%comment: auto-generated file %}
```
The first one will be included in a generated html as a comment, the second one won't.

**Comment by emillon** [comment]
As an extra use case: ocamlformat supports formatting mld files, but there's no way to specify options for formatting there (for example, to disable formatting of ocaml blocks within mld files). They could fit in a comment.
